### PR TITLE
P2P Promo and node-gyp fix for MacOS

### PR DESCRIPTION
**Pull request by AdrianAndersen**
- chore: create promo mail and sms
- deps: use a newer version of fsevents to support latest Xcode version
=> ref: https://github.com/nodejs/node-gyp/issues/1917#issuecomment-549839169
- 0.5.46

### CODE REVIEW

File: src/email/email-settings.ts [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const EMAIL_SETTINGS: any = {
   subjects: {
     reminder: {
       rent: [
-        "Påminnelse om tilbakelevering av bøker",
+        "Unikt tilbud til deg som skal begynne i VG2 ST på Ullern videregående skole til høsten",
```
Comment (LarsSelbekk) [inline src/email/email-settings.ts:23]: Does this not sound a little spammy?

**Review by LarsSelbekk — COMMENTED** [review]
Some small typos only

File: src/sms/sms-settings.ts [hunk 2]
```diff
@@ -21,7 +21,7 @@ export const SMS_SETTINGS = {
         ""
       ],
       rent: [
-        "",
+        "Hei, Boklisten åpner opp for at du kan få VG2-bøker allerede nå, hvis du bestiller bøkene i løpet av uka. Se informasjon vi har sendt på mail idag. Mvh, Boklisten.no",
```
[the remark this answers is not in the document]
Comment (AdrianAndersen) [reply src/sms/sms-settings.ts:24]: already sent 🐍

**Review by LarsSelbekk — APPROVED** [review]
Already sent 🤷‍♂️

**Comment by AdrianAndersen** [comment]
[Utsending av Promo-mail og Promo-SMS](https://trello.com/c/LYIzd4Ok/670-utsending-av-promo-mail-og-promo-sms)

**Pull request merged**